app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -405,4 +405,10 @@
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
 
+  <!-- RevisionCardFragment -->
+  <dimen name="revision_card_fragment_layout_min_width">512dp</dimen>
```

Comment:
As per previous comment, please rename to `revision_card_fragment_layout_tablet_min_width` so as to explicitly mention that its for tablet dimens. Ditto for other files.

Reply:
Sorry, Ig I missed this one. Now I have changed it PTAL